src/StackExchange.Redis/Message.cs

```diff
@@ -704,11 +757,9 @@ internal void SetRequestSent()
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         internal void SetWriteTime()
         {
-            if ((Flags & NeedsAsyncTimeoutCheckFlag) != 0)
-            {
-                _writeTickCount = Environment.TickCount; // note this might be reset if we resend a message, cluster-moved etc; I'm OK with that
-            }
+            _writeTickCount = Environment.TickCount; // note this might be reset if we resend a message, cluster-moved etc; I'm OK with that
```

Comment:
This will incur more overhead every call - do we need to raise it to every time?

Reply:
It was earlier being used every time for an async request.  https://github.com/StackExchange/StackExchange.Redis/blob/56ac9a21174684f30b8aa333f8069fdca77ed4e7/src/StackExchange.Redis/Message.cs#L749

I had to modify it so that sync requests also has the write time set. So that I can leverage it to figure out how long a sync message has been waiting, and if it's long enough time it out from the retry queue
            int millisecondsTaken = unchecked(Environment.TickCount - message.GetWriteTime());